techAssignment/src/App.vue

```diff
@@ -1,78 +1,107 @@
 <script>
+import InputCommand from "./components/InputCommand.vue";
+import CommandHistory from "./components/CommandHistory.vue";
+
 export default {
+  name: "App",
+
   data() {
     return {
-      count: 0,
+      commandInput: "",
+      commandInputArray: [],
+      arrayValues: [],
+      keyValue: {},
     };
   },
+
+  methods: {
+    renderInput() {
+      var commandInputArray = this.commandInput.split(" ");
+
+      switch (commandInputArray[0]) {
+        case "SET":
+          // Input must contain 3 words
+          if (commandInputArray.length != 3) {
+            this.data("ERROR");
```

Comment:
Could you help explain more what `this.data(string)` do? I don't see the method `data` defined in this component. Should it be `this.loadArrayValues("ERROR");`?